suite.yml

```diff
@@ -16,15 +16,15 @@ section:
         url: https://github.com/cyberark/conjur-oss-helm-chart
         description: Helm chart for deploying Conjur OSS.
         upgrade_url: https://github.com/cyberark/conjur-oss-helm-chart/tree/master/conjur-oss#upgrading-modifying-or-migrating-a-conjur-oss-helm-deployment
-        version: v2.0.2
+        version: v2.0.3
```

Comment:
Looking at the changelog, this probably should have been v2.1.0, due to the OCP addition.

Reply:
ah, you're right :/